cli/ps.go

```diff
@@ -10,17 +12,32 @@ import (
 
 func init() {
 	register("ps", runPs, `
-usage: flynn ps
+usage: flynn ps [-a]
 
 List flynn jobs.
 
+Options:
+  -a, --all      Show all jobs (default is just running and pending)
+
 Example:
 
 	$ flynn ps
-	ID                                      TYPE  RELEASE
-	flynn-bb97c7dac2fa455dad73459056fabac2  web   b69d7fb5308a4684a09b160b82d267ec
-	flynn-c59e02b3e6ad49809424848809d4749a  web   b69d7fb5308a4684a09b160b82d267ec
-	flynn-46f0d715a9684e4c822e248e84a5a418  web   b69d7fb5308a4684a09b160b82d267ec
+	ID                                         TYPE  STATE    CREATED         RELEASE
+	ab14754c-73b7-4212-a6d9-73b825587fd2       web   pending  2 seconds ago   1b1db8ef-ba4d-4314-85c1-d5895a44b27e
+	6ec25d6e-2985-4807-8e64-02dc23c348bc       web   pending  7 seconds ago   1b1db8ef-ba4d-4314-85c1-d5895a44b27e
+	host-f25797dc-c956-4337-89af-d49eff50f58e  web   up       14 seconds ago  1b1db8ef-ba4d-4314-85c1-d5895a44b27e
+
+	$ flynn ps --all
+	ID                                         TYPE  STATE    CREATED             RELEASE
+	ab14754c-73b7-4212-a6d9-73b825587fd2       web   pending  2 seconds ago       1b1db8ef-ba4d-4314-85c1-d5895a44b27e
+	6ec25d6e-2985-4807-8e64-02dc23c348bc       web   pending  7 seconds ago       1b1db8ef-ba4d-4314-85c1-d5895a44b27e
+	host-f25797dc-c956-4337-89af-d49eff50f58e  web   up       14 seconds ago      1b1db8ef-ba4d-4314-85c1-d5895a44b27e
+	host-ef6249e3-b463-4fea-a7dd-b1302872f821  web   down     20 seconds ago      1b1db8ef-ba4d-4314-85c1-d5895a44b27e
+	host-cef95d8c-a632-4ae6-8a57-e13cbc24afa9  web   down     25 seconds ago      1b1db8ef-ba4d-4314-85c1-d5895a44b27e
+	host-95747b4f-fdcd-4c44-ab72-b3d9609668e4  run   down     54 seconds ago      cd698657-2955-4fa4-bc2f-8714b218a7a2
+	host-e8b4f9be-e422-481f-928c-4c82f0bb5e8b  web   down     About a minute ago  cd698657-2955-4fa4-bc2f-8714b218a7a2
+	host-feaff633-a37b-4565-9ade-24bae0cfae03  web   down     About a minute ago  cd698657-2955-4fa4-bc2f-8714b218a7a2
+	host-d84dc657-83b8-4a62-aab8-ad97bb994761  web   down     2 minutes ago       cd698657-2955-4fa4-bc2f-8714b218a7a2
```

Comment:
Does it make sense to sort in this direction? If the list scrolls off the screen, you'd have to scroll up to see the most relevant data.